guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,701 @@
+that it should not be used on tasks where the data distribution is not vertical
+mirror invariant.
+An example of a dataset where this occurs is MNIST hand written digits.
+Flipping a `6` over the
+vertical axis will make the digit appear more like a `7` than a `6`, but the
+label will still show a `6`.
+"""
+random_flip = keras_cv.layers.RandomFlip()
+augmenters = [random_flip]
+
+image_batch = random_flip(image_batch)
+keras_cv.visualization.plot_image_gallery(
+    image_batch.to_tensor(),
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+"""
+Half of the images have been flipped!
+
+The next augmentation we'll use is `RandomCropAndResize`.
+This operation selects a random subset of the image, then resizes it to the
+provided target size.
+By using this augmentation, we force our classifier to become spatially invariant.
+Additionally, this layer accepts an `aspect_ratio_factor` which can be used to
+distort the aspect ratio of the image.
+While this can improve model performance, it should be used with caution.
+It is very easy for an aspect ratio distortion to shift a sample too far from
+the original training set's data distribution.
+Remember - the goal of data augmentation is to produce more training samples
+that align with the data distribution of your training set!
+
+`RandomCropAndResize` also can handle `tf.RaggedTensor` inputs.  In the
+CalTech101 image dataset images come in a wide variety of sizes.
+As such they cannot easily be batched together into a dense training batch.
+Luckily, `RandomCropAndResize` handles the Ragged -> Dense conversion process
+for you!
+
+Let's add a `RandomCropAndResize` to our set of augmentations:
+"""
+crop_and_resize = keras_cv.layers.RandomCropAndResize(
+    target_size=IMAGE_SIZE,
+    crop_area_factor=(0.8, 1.0),
+    aspect_ratio_factor=(0.9, 1.1),
+)
+augmenters += [crop_and_resize]
+
+image_batch = crop_and_resize(image_batch)
+keras_cv.visualization.plot_image_gallery(
+    image_batch,
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+"""
+Great!  We are now working with a batch of dense images.
+Next up, lets include some spatial and color-based jitter to our training set.
+This will allow us to produce a classifier that is robust to lighting flickers,
+shadows, and more.
+
+There are limitless ways to augment an image by altering color and spatial
+features, but perhaps the most battle tested technique is
+[`RandAugment`](https://arxiv.org/abs/1909.13719).
+`RandAugment` is actually a set of 10 different augmentations:
+`AutoContrast`, `Equalize`, `Solarize`, `RandomColorJitter`, `RandomContrast`,
+`RandomBrightness`, `ShearX`, `ShearY`, `TranslateX` and `TranslateY`.
+At inference time, `num_augmentations` augmenters are sampled for each image,
+and random magnitude factors are sampled for each.
+These augmentations are then applied sequentially.
+
+KerasCV makes tuning these parameters easy using the `augmentations_per_image`
```

Comment:
Can you offer any advice on how to pick these parameters?

Reply:
To be honest I just look at it and decide if it looks enough like the originals